hazelcast-sql/src/main/java/com/hazelcast/sql/impl/calcite/parse/UnsupportedOperationVisitor.java

```diff
@@ -165,6 +165,10 @@ private void processSelect(SqlSelect select) {
         if (select.getGroup() != null && select.getGroup().size() > 0) {
             throw unsupported(select.getGroup(), "GROUP BY");
         }
+
+        if (select.getFetch() != null) {
```

Comment:
We need a unit test to cover this case in `ParserOperationsTest`. Something like:
```
@Test
    public void testUnsupportedLimit() {
        checkFailure(
            "SELECT a FROM t LIMIT 10",
            "LIMIT is not supported"
        );
    }
```

Reply:
Fixed.